spark_pipeline_framework/pipelines/framework_param_pipeline.py

```diff
@@ -84,7 +84,7 @@ def _transform(self, df: DataFrame, response: Dict[str, Any]) -> Any:  # type: i
                     if isinstance(transformer, FrameworkParamTransformer):
                         response = transformer.transform(dataset=df, response=response)
                     elif isinstance(transformer, FrameworkTransformer):
-                        response = transformer.transform(dataset=df)  # type: ignore
+                        _ = transformer.transform(dataset=df)
```

Comment:
@Harisai-Veeramallu here, if this variable `_` is used for ignoring the return value of this `FrameworkTransformer`'s `.tranform()` method call, then why don't we just remove the whole `elif` block?
IF we must have this elif logical block, then we could keep it as it was with the `response = `?

Reply:
Actually, this part is to provide support to run the regular `FrameworkPipeline` transformers in `FrameworkParamPipeline`. As the `FrameworkPipeline` takes only `df` as input and returns `df` as response, we have to handle it separately using `elif`. And as far as the `response` is concerned, the `transformer.transform()` in line 87 returns a `DataFrame`, and we do not pass on `DataFrame` in out new flow, we pass on the `response` which is of type `Dict[str, Any]`.

Reply:
Ok, good for - we still need to support this elif block,
then what's harm to leave that `response = ` as it was in line 87 to support the current existing DataFrame approach?
it may be a good idea to create another variable with different name then?

Reply:
We actually need to execute the `transform()` without capturing the output of that function and we do not want to modify the `response` variable in this case.